sandboxes/crates.io/Dockerfile

```diff
@@ -5,14 +5,18 @@ RUN apt-get update && \
         curl \
         wget \
         git \
-        python
+        python \
+		sudo
 
 COPY analyze.py /usr/local/bin/
 RUN chmod 755 /usr/local/bin/analyze.py
 RUN mkdir -p /app
 WORKDIR /app
 RUN cargo init
 
+# Configure sudo for passwordless execution
+RUN echo "ALL ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
+
```

Comment:
This should go under the `apt-get install` (and the other Dockerfiles too)

Reply:
Done